### ISSUE TITLE
Off state of toggle (and small toggle) lacks 3:1 contrast with adjacent color (background)

### ISSUE THREAD

**Issue by mbgower** [carbon-design-system/carbon#3614]
## Off state of toggle lacks 3:1 contrast with adjacent color ( either internal circle or background)

-->

## Environment
v10, react, theme
> Operating system
Windows 7

> Browser
Firefox and Chrome

## Detailed description
There are different implementations of the toggle, but all fail contrast for the 'off' state. 

> What did you expect to happen?
The off state should meet the minimum contrast. 

> What WCAG 2.1 checkpoint does the issue violate?
[Non-text Contrast](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html)

## Steps to reproduce the issue

1. Step one
2. Step two
3. Step three
4. etc.

> Please create a reduced test case in CodeSandbox
>
> - Style and vanilla JS:
>   https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/components/examples/codesandbox
> - React:
>   https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/react/examples/codesandbox

## Additional information
![image](https://user-images.githubusercontent.com/14298245/62251135-a4f3e800-b3a3-11e9-82a9-6c8aaa4b9983.png)
Storybook Toggle on (sufficient)
![image](https://user-images.githubusercontent.com/14298245/62251155-afae7d00-b3a3-11e9-8a98-b00e75eb4129.png)
Storybook Toggle off (insuficient 2.5:1 between oval and lighter gray of circle)
![image](https://user-images.githubusercontent.com/14298245/62251177-c0f78980-b3a3-11e9-9bd0-1912e59a33f2.png)
Theme on and off toggle (same failure ratio as first screen shot)
![image](https://user-images.githubusercontent.com/14298245/62251216-dec4ee80-b3a3-11e9-9de3-181a5b84278a.png)
V10 showing on and insufficient off toggle ( contrast for 1.8:1)
![image](https://user-images.githubusercontent.com/14298245/62252119-c1455400-b3a6-11e9-9f17-5b2a6440c305.png)
v10 Toggle small, with similar challenges for the 'off' toggle

### Notes
In a situation where ALL the toggle versions contained a visible text state (On/Off) the non-text contrast could likely be excused since the visual cue would be redundant to the text and thus not _required_ ("Visual information required to identify user interface components and states...)
That is, with a text state of "off" visible, the user no longer needs to discern the state of the toggle. However, this is a fairly easy issue to resolve -- just ensure the toggle border is 3:1 against the white background (for the v10 version) and that the circle is a lighter gray (or white) so that it is visible.

**Comment by asudoh** [comment]
CC @carbon-design-system/design just to see if the team has had a discussion on this topic.

**Comment by shixiedesign** [comment]
Waaaaait I know this issue must be fixed this sprint but y’all don’t know which colors are accessible yet with the new color palette getting implemented. Hang on till tomorrow.

**Comment by shixiedesign** [comment]
### Toggle final spec (updated according to Anna's comment below)
Please note: 
- Hover state should look the same as default. 
- Disabled state has no change. 

```
// on state

pill: $support-02
circle: $icon-03

// off state

pill: $ui-04
cirlce: $icon-03
```

![image](https://user-images.githubusercontent.com/15144993/64751692-90e2f080-d4e2-11e9-9bd3-fe6f25e22b3a.png)

### Focus styling update

Additional change is adding a white inside border to focus state, so the focus border is 3:1 accessible against component & background. Reference the same treatment in primary button. 

![image](https://user-images.githubusercontent.com/15144993/64751710-9fc9a300-d4e2-11e9-9e03-72372c86b5d5.png)

**Comment by aagonzales** [comment]
### No hover
I don't think it should have a hover. Most toggles ive seen in the wild don't have a hover. Also similar control in Carbon, like check box and radio, don't have hovers

### Focus
Agreed, I like the focus options you chose.
